File: TIOJ/1506/main.cpp
#include<bits/stdc++.h>
using namespace std;

signed main(){
	float a1, a2, b1, b2;
	cin>>a1>>a2>>b1>>b2;
	float x, y;
	x = -(b1-b2)/(a1-a2)*100;
	y = (a1*b2-a2*b1)/(a1-a2)*100;
	cout<<fixed<<setprecision(2);
	cout<<round(x)/100<<"\n"<<round(y)/100;
	return 0;
}

File: TIOJ/1506/main.py
a1 = int(input())
a2 = int(input())
b1 = int(input())
b2 = int(input())


x = -(b1-b2)/(a1-a2)*100
y = (a1*b2-a2*b1)/(a1-a2)*100
print(f"{round(x)/100:0.2f}")
print(f"{round(y)/100:0.2f}")

